examples/mapviewer.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
Should maybe be `#!/usr/bin/env python3` to make it work intuitively (e.g. the current implementation will break in a virtualenv).

Reply:
Done.